### ISSUE TITLE
Make the Verse block the same as "Paragraph" but exclusively with line breaks

### ISSUE THREAD

**Issue by unpack5** [WordPress/gutenberg#15509]
All this is rooted in one decision: when Wordpress decided to assign "new paragraph" to Enter.

All I want is to be able to write as I did, where Enter inserts a line break and text/formatting looks normal.

I'm very sorry to say this but I'm disgusted by the newer versions of the text editor. But after much reading I found out I need Verse blocks to be able to write as I did. I had to jump two versions forward to access the feature.

Then it turns out that, although now I can insert line break with Enter, Verse is the same as Preformatted. So if I wanted to do this, I could have just stayed with the old-old editor.

I'm frustrated that I had to dig this far just to find I can't do what I could do with this software a few years ago. Please make Verse block what its name implies, a properly looking (no weird font, no background box), formattable text type (so basically what you have in Paragraph) where you can enter a line break simply by hitting Enter. No weird key shortcuts to achieve this, just what an author needs.

**Comment by youknowriad** [comment]
Hi There! sorry for the issues your having. The verse block is not meant to be used for writing normal content. As it's name suggests it's for verses/poetry...

Can you clarify more why the paragraph block is not working for you as expected?

What issues are you having when you hit "Enter"?

**Comment by swissspidy** [comment]
> I'm frustrated that I had to dig this far just to find I can't do what I could do with this software a few years ago.

In the Classic Editor, hitting Enter created a new paragraph. In the new editor, the behavior is exactly the same. If you want simple line breaks, use Shift+Enter.

This is a common practice in many many text editors out there.

**Comment by unpack5** [comment]
> The verse block is not meant to be used for writing normal content. As it's name suggests it's for verses/poetry...

I was actually trying to write poetry in it. But a monospace font and a code block-like gray frame is very different from the atmosphere of a literary work. It matches much more something like programming. Which is not surprising, since the Verse block now seems to be doing the same exact thing as a Preformatted block.

As far as I can see, there is no need to have two Preformatted block types under different names, and Verse block is supposed to serve a poetic purpose. Poems in printed books are mostly set in a variable-width serif font, have formatting like italics and involve a lot of line breaks but considerably less paragraphs. I believe the Verse block should serve that use case.

 > Can you clarify more why the paragraph block is not working for you as expected?

I wanted to write a poem, with short lines and a *lot* of line breaks. Having to press Shift+Enter at every line was cumbersome and I eventually lost inspiration and never finished the poem. I don't think that effortless creation should be subordinated to technicalities. You could say that if I want this, I could use a Word document, but then I don't see why we have something called "Verse block".

> In the Classic Editor, hitting Enter created a new paragraph. In the new editor, the behavior is exactly the same. If you want simple line breaks, use Shift+Enter.
>
> This is a common practice in many many text editors out there.

I my be mistaken but I remember a time when I didn't have to meddle with Shift+Enter in Wordpress to make a simple line break. It might have been a long time ago. Nevertheless, as I said, I think verse block should serve the purpose that's in its name. If I want a new paragraph, I can press Enter twice, without having to distract my attention with a key combo.

**Comment by youknowriad** [comment]
> But a monospace font and a code block-like gray frame is very different from the atmosphere of a literary work

What theme are you using? 
The font and the background of any block can be changed by themes. For example in Twenty Nineteen, the verse block doesn't use a monospace font and there's no border/background.

**Comment by unpack5** [comment]
> What theme are you using?

I think it's Twenty-twelve. Can the theme be updated?

**Comment by youknowriad** [comment]
Sorry for the late response. So yes, the theme can be updated if the authors decide it's something that's valuable for all the theme users. If you think so, I suggest you create a trac ticket here https://core.trac.wordpress.org/ and contributors there will have better answers for you. Thanks

**Comment by webambience** [comment]
No one should have to go as long as me without learning that holding down the shift key and hitting enter will get you a single space vs a paragraph.  Then I actually learned at first that I had to click them at the same time lol, which was tedious in a different way.  At any rate, hopefully someone out there who hates that wordpress does new paragraph on enter, just hold down the shift key and hit enter for a single space.   By the way, the "verse " block is pretty ugly at least with the theme I am using.  I can't get poetry to look right on my website, the formatting gets stripped out and it can't get straight vs smart quotes right.
